docs/network-flow-visibility.md

```diff
@@ -710,34 +745,53 @@ serviceTemplates:
           port: 9000
 ```
 
-This service is also used by the Flow Aggregator and Grafana. If you update the
-HTTP port, please update `url` in [datasource_provider.yml][grafana_datasouce_provider_yaml].
-If you update the TCP port, please update `jsonData.port` in [datasource_provider.yml][grafana_datasouce_provider_yaml]
-and `databaseURL` in the [Flow Aggregator Configuration](#configuration-1).
+This service is used by the Flow Aggregator and Grafana.
+
+- If you have changed the HTTP port, please update the `url` of a resource of name `grafana-datasource-provider` in `flow-visibility.yml`.
```

Comment:
Maybe we need to keep consistency in whether we need a colon after "a resource of name"?

Reply:
I will also join the chorus and ask to change 'resource of name' to 'ConfigMap named' at lines 750 and 753